taca/backup/backup.py

```diff
@@ -279,11 +314,19 @@ def pdc_put(cls, run):
                 continue
             flag = open(run.flag, 'w').close()
             with filesystem.chdir(run.path):
+                if bk.file_in_pdc(run.zip_encrypted, silent=False) or bk.file_in_pdc(run.dst_key_encrypted, silent=False):
+                    logger.warn("Seems like files realted to run {} already exist in PDC, check and cleanup".format(run.name))
+                    bk._clean_tmp_files([run.flag])
+                    continue
+                logger.info("Sending file {} to PDC".format(run.zip_encrypted))
                 if bk._call_commands(cmd1="dsmc archive {}".format(run.zip_encrypted), tmp_files=[run.flag]):
+                    time.sleep(15) # give some time just in case 'dsmc' needs to settle
```

Comment:
Hej @vezzi , it should be "ready" always. The reason why I kept that is just me being paranoid 😅 . So when a `dsmc` is executed it connects to server (PDC) and what ever we asked to do and ends the connection. 

Until it ends the connection with the server it wont return exit code (or thats how I assume it be by using till now) and so the next python line will not executed. But like I said earlier I am just being paranoid and want the processor to catch some breath for next sprint 😉 

P.S: The word "ready" might have given a wrong idea here, shall I change the comment so it doesn't mislead ?